src/cts/src/HTreeBuilder.cpp

```diff
@@ -347,7 +347,9 @@ void HTreeBuilder::computeLevelTopology(unsigned level,
   logger_->report("    Sub-region size: {:.4f} X {:.4f}", width, height);
 
   const unsigned minLength = minLengthSinkRegion_ / 2;
-  unsigned segmentLength = std::round(width / (2.0 * minLength)) * minLength;
+
+  unsigned segmentLength
+      = minLength == 0 ? 0 : std::round(width / (2.0 * minLength)) * minLength;
   if (isVertical(level)) {
     segmentLength = std::round(height / (2.0 * minLength)) * minLength;
```

Comment:
It would be a error if it ever were zero and probably nothing else would work.  Why isn't it an error on 354 as well?

Reply:
Agreed on both points -- I'll investigate more deeply than just working around the exception we saw, please don't merge yet. I'm hoping we can get to the bottom of why the logger error fails on the regression tests tomorrow. (Will try to convert this to "draft" mode to indicate we're going to investigate more.)